test/Microsoft.AspNet.Mvc.Xml.Test/XmlDataContractSerializerInputFormatterTest.cs

```diff
@@ -485,5 +502,16 @@ private InputFormatterContext GetInputFormatterContext(byte[] contentBytes, Type
             httpContext.SetupGet(c => c.Request).Returns(request.Object);
             return httpContext.Object;
         }
+
+        private class TestXmlDataContractSerializerInputFormatter : XmlDataContractSerializerInputFormatter
+        {
+            public int calledCount = 0;
```

Comment:
nit: rename this to something like `CreateSerializerCallCount` (`formatter.CreateSerializerCallCount` feels better than `formatter.callCount`)